mlflow/gateway/__init__.py

```diff
@@ -1,3 +1,9 @@
+"""
+.. warning::
+
+    `mlflow.gateway` is deprecated and will be replaced by the deployments API in a future release.
```

Comment:
The deployments API will be released at the same time that gateway is deprecated. Can we rephrase?